examples/index.md

```diff
@@ -5,7 +5,10 @@ This folder contains examples of pre-defined configuration files for common scen
 | Config file name         | Public IP | Storage | VNET/Subnet | Azure Monitor | Alerting | Lustre | Slurm Accounting |
 |--------------------------|-----------|---------|-------------|---------------|----------|--------|------------------|
 | minimum_public_ip.yml    | Yes       | ANF     | Create      | No            | No       | No     | No               |
-
+| qinstall-config.yml    | Yes       | ANF     | Create      | No            | No       | No     | No               |
 
 ## minimum_public_ip.yml
 This configuration file is defining the minimum required to build an `azhop` environment when public IP is allowed.
+
+## qinstall-config.yml
+This configuration file is defining the minimum required to quickly deploy `azhop` environment.
```

Comment:
Please describe that it comes with the quick install or add a link to the tutorial page

Reply:
I added the link to follow the guide.